metadata-service/configuration/src/main/resources/application.yml

```diff
@@ -45,6 +45,8 @@ authorization:
     cachePolicyFetchSize: ${POLICY_CACHE_FETCH_SIZE:1000}
   # Enables authorization of reads, writes, and deletes on REST APIs. Defaults to false for backwards compatibility, but should become true down the road
   restApiAuthorization: ${REST_API_AUTHORIZATION_ENABLED:false}
+  search:
+    enabled: ${SEARCH_AUTHORIZATION_ENABLED:true}
```

Comment:
and we want this on by default for everyone right? just calling it out as a big change. we could always start with this turned off while we validate things

Reply:
I will flip this however there is a default policy which allows everyone read. The risk is some new bug introduced in the filter logic.